src/js/components/InfiniteScroll/InfiniteScroll.js

```diff
@@ -270,7 +323,11 @@ const InfiniteScroll = ({
         key="below"
         ref={belowMarkerRef}
         flex={false}
-        height={`${replace ? (lastPage - endPage) * pageHeight : 0}px`}
+        height={`${
+          replace && pageHeight && !onMore
```

Comment:
Would a comment as to why `!onMore` be helpful here?

Reply:
ill add comment above. but !onmore will prevent the height of the bottom marker from being calculated when onmore and replace are both available. It's not possible, at least to me, for us to know the correct amount of pages needed for this calculation. I think its safe to assume that if a user passes onMore, that behavior should take precedence over replace.

Reply:
I was thinking that sort of rationale would be worthwhile to leave for future contributors.